src/style.css

```diff
@@ -2385,7 +2389,11 @@ html {
   padding: 8px 15px;
   text-align: center;
   vertical-align: middle;
-  touch-action: manipulation;
+  touch-action: manipulation;  
+}
+
+.normalcase {
+  text-transform: none
 }
```

Comment:
See above comment - This is not needed as tailwind has a build in class
https://tailwindcss.com/docs/text-transform

```suggestion
```